model_test.go

```diff
@@ -723,6 +723,7 @@ remote-applications:
   spaces:
     spaces: []
     version: 1
+  status: null
```

Comment:
let's set a status - status would not be empty, it would start out as something like "unknown" like we do for units